src/server/page_server.cpp

```diff
@@ -210,7 +210,7 @@ page_server::connection_handler::receive_oldest_active_mvccid (tran_server_conn_
 {
   assert (m_server_type == transaction_server_type::PASSIVE);
 
-  const auto oldest_mvccid = *reinterpret_cast<const MVCCID *const> (a_sp.pull_payload().c_str());
+  const auto oldest_mvccid = *reinterpret_cast<const MVCCID *> (a_sp.pull_payload().c_str());
```

Comment:
Just out of curiosity, would you tell me why this is better? Thanks.

Reply:
It is just a warning reported by gcc9.
Basically, it had no effect as you want to convert  some memory address to some data type. It is only then that you either assign the result of that conversion to a variable with either const-volatile qualifiers. The conversion itself has no role in that assignment. 

For context, this article explains some aspect which, I think, is also involved in the above:

http://www.dansaks.com/articles/2000-02%20Top-Level%20cv-Qualifiers%20in%20Function%20Parameters.pdf